src/app/report-dashboard/report-dashboard.component.html

```diff
@@ -21,23 +21,24 @@ <h1 class="w-min whitespace-nowrap text-3xl font-bold">Rapportage dashboard</h1>
     <div echarts [options]="mostUsagesPieOptions" class="w-3/6 h-[400px] border-2 border-secondary-purple-light p-4 rounded-lg shadow-lg"></div>
     <div echarts [options]="leasUsagesPieOptions" class="w-3/6 h-[400px] border-2 border-secondary-purple-light p-4 rounded-lg shadow-lg"></div>    
   </section>
-  <div echarts [options]="heatmapOptions" class="h-[500px] border-2 border-secondary-purple-light p-4 rounded-lg shadow-lg"></div>
-  <div class="flex w-full border-2 border-secondary-purple-light p-10 rounded-lg shadow-lg">
+  <div echarts [options]="heatmapOptions" class="h-[500px] border-2 border-secondary-purple-light p-4 rounded-lg shadow-lg overflow-x-auto"></div>
```

Comment:
De cijfers bij de heatmap lijken een beetje te veel

Reply:
Wat bedoel je precies? Ik kan de zoom in het begin ook nog vergroten dat er nog minder wordt laten zien maar dat je nog wel uit kan zoomen

Reply:
Als je heel ver uit zoomt dan worden de cijfers zo klein dat het meer cluttering eruit ziet dan dat het echt betekenis heeft, misschien op een x aantal zoom level de cijfers uitzetten?